transient_replication_ring_test.py

```diff
@@ -226,13 +227,12 @@ def test_bootstrap_and_cleanup(self):
         # Every value should be replicated exactly 2 times
         self.check_replication(sessions, exactly=2)
 
-    @flaky(max_runs=1)
```

Comment:
Yes, I was talking about this one but please, remove it for all tests in the class. It is the same useless and don't seem right to me to leave it there. Thanks